include/openrave/geometry.h

```diff
@@ -491,11 +491,10 @@ class RaveTransform
         return !operator==(right);
     }
 
-    inline bool Compare(const RaveTransform<T>& rhs, T epsilon=1e-7) const {
-        T e1 = (rot-rhs.rot).lengthsqr4();
-        T e2 = (rot+rhs.rot).lengthsqr4();
-        T e = e1 < e2 ? e1 : e2;
-        return RaveSqrt((trans-rhs.trans).lengthsqr3() + e) <= epsilon;
+    /// \brief return true if any element of transform is different by mor ethan epsilon, otherwise false.
+    inline bool CompareTransform(const RaveTransform<T>& rhs, T epsilon) const {
```

Comment:
@ziyan @rdiankov Is it intentional that we don't account for the fact that `quat` and `-quat` represent the exact same orientation here?